resources/quality/ultimaker_s3/um_s3_aa0.4_um-tough-pla_0.15mm.inst.cfg

```diff
@@ -37,9 +37,9 @@ jerk_roofing = =jerk_wall_0
 jerk_topbottom = =jerk_wall
 jerk_wall = =jerk_infill
 jerk_wall_0 = =max(30, speed_wall_0/2)
-machine_nozzle_cool_down_speed = 1,3
-machine_nozzle_heat_up_speed = 1,9
-material_extrusion_cool_down_speed = 0,7
+machine_nozzle_cool_down_speed = 1.3
+machine_nozzle_heat_up_speed = 1.9
+material_extrusion_cool_down_speed = 0.7
 material_max_flowrate = 14
```

Comment:
material_print_temperature = =default_material_print_temperature +10

Reply:
Temperature should be the same as [um_s3_aa0.4_tough-pla_0.15mm](https://github.com/Ultimaker/Cura/blob/main/resources/quality/ultimaker_s3/um_s3_aa0.4_tough-pla_0.15mm.inst.cfg)